bindings/python/iota_sdk/common.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from json import dumps, JSONEncoder
+from enum import Enum
+import humps
+
+
+def custom_encoder(func, *args, **kwargs):
+    """The routine of dump json string.
+    """
+    class MyEncoder(JSONEncoder):
+        """Custom encoder
+        """
+        # pylint: disable=too-many-return-statements
```

Comment:
such a weird lint: What are you supposed to do to fix it if you really wanted to?

Reply:
I guess the lint checker is just not really smart, for the example here they have a suggestion
https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-return-statements.html

Reply:
Feels like a really weird thing to be strongly opinionated about